test-case/check-smart-amplifier.sh

```diff
@@ -98,14 +98,16 @@ do
         file="$tmp_dir/tmp_wave_${fmt%_*}.wav"
         recorded_file="$tmp_dir/recorded_tmp_wave_${fmt%_*}.wav"
         wavetool.py -gsine -A0.8 -B"${fmt%_*}" -o"$file"
-        dlogc "aplay -D$pb_dev -r $pb_rate -c $pb_chan -f $fmt -d $duration -q $file &"
-        aplay -D"$pb_dev" -r "$pb_rate" -c "$pb_chan" -f "$fmt" -d "$duration" -v -q "$file" &
-        dlogc "arecord -D$cp_dev -r $cp_rate -c $cp_chan -f $fmt -d $duration -q $recorded_file"
-        arecord -D"$cp_dev" -r "$cp_rate" -c "$cp_chan" -f "$fmt" -d "$duration" -v -q "$recorded_file"
+        dlogc "arecord -D$cp_dev -r $cp_rate -c $cp_chan -f $fmt -d $duration -v -q $recorded_file &"
+        arecord -D"$cp_dev" -r "$cp_rate" -c "$cp_chan" -f "$fmt" -d "$duration" -v -q "$recorded_file" &
```

Comment:
To make sure no error is left behind ("green failures"), `wait` for all background commands at the end which collects their status. Especially commands like `arecord` or `aplay`; we _really_ don't want to miss any error from ALSA commands because these are audio tests! See example in recent commit fd545aefac3a:
```shell
timeout -k 3 10 some_backgrounded_command & somePID=$!
... test ... test .... test
wait $somePID || die "alsa_blabla failed"
```

Reply:
@marc-hb Yes, We are hiding the bug for release test with this case next week. But seems FW team find the root cause of the bug, so we don't need the workaround. I will come back later to apply your suggestion, as WOV case is first priority.